index.html

```diff
@@ -43,36 +43,36 @@ <h1 class="binabox">
       </div>
       <section id="catalog">
         <h2 class="catalog" >Catalog</h2>
-        <button class="catalog-button">&lt;</button>
-        <button class="catalog-button">&gt;</button>
-        <button class="catalog-button__see">See all</button>
+        <button class="catalog__button">&lt;</button>
+        <button class="catalog__button">&gt;</button>
+        <button class="catalog__button--see">See all</button>
```

Comment:
two classes here: "catalog__button" and "catalog__button--see"
first one is for block__element, second is basic class with modifier